docs-template/EXAMPLE_README.md.tpl

```diff
@@ -240,6 +240,14 @@ Bevy support for WebGPU is being worked on, but is currently experimental.
 
 To build for WebGPU, you'll need to disable default features and add all those you need, making sure to omit the `webgl2` feature.
 
+Also, given the fact that WebGPU depends on unstable APIs you will also need to pass the `web_sys_unstable_apis` flag to your builds. For example:
```

Comment:
```suggestion
WebGPU depends on unstable APIs so you will also need to pass the `web_sys_unstable_apis` flag to your builds. For example:
```

Just a minor nitpick, de-fluffing this sentence a bit by removing the "double also."

Reply:
Applied in 58c9cac24823b702274e0822e67a1211bb868f2c